src/server/handshake.js

```diff
@@ -9,10 +9,14 @@ module.exports = function (client, server, { version }) {
     client.protocolVersion = packet.protocolVersion
 
     if (version === false || version === undefined) {
-      if (require('minecraft-data')(client.protocolVersion)) {
-        client.version = client.protocolVersion
+      if (client.protocolVersion === 5) { // The first snapshot versions of 1.8 uses the 1.7 protocol id, after 1.8 each snapshot has a different protocol id.
+        client.version = '1.7.10'
```

Comment:
I know this is the only way to make it works, but this is no longer always a protocolversion anymore, which is odd